README.md

```diff
@@ -342,6 +342,7 @@ Currently, the following report formats are supported (reporter names are case-i
 * [NOTICE](http://www.apache.org/dev/licensing-howto.html) file in two variants
   * List license texts and copyrights by package (`-f NoticeByPackage`)
   * Summarize all license texts and copyrights (`-f NoticeSummary`)
+* [SPDX Document](https://github.com/spdx/spdx-spec/tree/v2.2) (`-f SpdxDocument`)
```

Comment:
I'd prefer to include the supported version in the name, but generically link to https://spdx.dev/specifications/, like

    [SPDX Document, version 2.2](https://spdx.dev/specifications/)

If we'd support more versions in the future, then these only need to be added to the name.

Reply:
Also, are you planning to add reporter-specific docs, like we have for the `AntennaAttributionDocumentReporter`? If not, we should mention here at least that only JSON and YAML are supported, but neither tag-value or RDF formats.

BTW, I believe we need the reporter-specific docs, as there are several options to `SpdxDocumentReporter` that deserve some explanation to the end user.

Reply:
Sure, I'll add a reporter specific documentation in a following PR.